opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -467,7 +471,7 @@ namespace Opm {
                 updateWellStatus( *well , currentStep , status );
                 if (well->setProductionProperties(currentStep, properties))
                     m_events.addEvent( ScheduleEvents::PRODUCTION_UPDATE , currentStep);
```

Comment:
Should not this (and injection, polymer) update also insert a well production update event?